modules/ob-installing-builds-with-operator.adoc

```diff
@@ -19,19 +19,36 @@ You can install the {builds-operator} from the OperatorHub by using the {ocp-pro
 
 . On the *Install Operator* page:
 
-.. Check that the *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.
+.. Check that the *Installation Mode* is set to *All namespaces on the cluster (default)*. This mode installs the Operator in the default `openshift-builds` namespace to watch and be made available to all namespaces in the cluster.
 
-.. Check that the *Installed Namespace* is set to `openshift-operators` by default.
+.. Check that the *Installed Namespace* is set to `openshift-builds` by default.
 
 .. Select *Automatic* for the *Approval Strategy*. This ensures that the future upgrades to the Operator are handled automatically by the Operator Lifecycle Manager (OLM). If you select the *Manual* approval strategy, OLM creates an update request. As a cluster administrator, you must then manually approve the OLM update request to update the Operator to the new version.
 
 .. Select an *Update Channel*:
 
 ** The *Update Channel* is set to *latest* by default. The latest channel enables installation of the most recent stable version of the {builds-operator}.
-** To install a specific version of the {builds-operator}, cluster administrators can use the corresponding `openshift-builds-<version>` channel. For example, to install the {builds-operator} version `0.11.0`, you can use the `openshift-builds-0.11.0` channel.
+** To install a specific version of the {builds-operator}, cluster administrators can use the corresponding `builds-<version>` channel. For example, to install the {builds-operator} version `1.1`, you can use the `builds-1.1` channel.
 
 . Click *Install*.
 
 .Verification
 
-* On the *Installed Operators* page, verify that the {builds-operator} is listed and its *Status* is set to *Succeeded* to confirm successful installation.
+After installing the Builds for Red Hat OpenShift Operator, you must verify that the required resources are created to ensure proper functioning of the controllers.
+
+.Procedure
+. In the *Administrator* perspective of the web console, select the project as openshift-builds.
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+
+. Click the *All instances* tab. 
+
+. On the *All instances* tab, verify that the `OpenShiftBuild` & `ShipwrightBuild` instances are listed. Alternatively, you can also click the Open Shift Build and Shipwright Build tabs.
```

Comment:
"Alternatively, you can also click the Open Shift Build and Shipwright Build tabs." This not valid for 1.2, since these tabs are removed in 1.2.